techmatters/serverless
# CHI-2790: Alert service user for unsupported media

## Description

- New feature: This PR implements a webhook that would send a message to callers whenever they send unsupported media. This can also cover the custom channel [ticket](https://tech-matters.atlassian.net/browse/CHI-2841) since the message sent out is generic as we cannot determine the media type for the unsupported message. 

- Next Action: To implement a unit test for the newly implemented `serviceConversationListener` webhook.



### Checklist

### Other Related Issues

None

[CHI-2790](https://tech-matters.atlassian.net/browse/CHI-2790)

### Verification steps


- Start a conversation (WhatsApp, Line, Telegram, etc)
- Send an unsupported media file (location in WhatsApp, Stick or image in Line/Telegram)
- Verify that you get an automatic response notifying you about the reaction not delivered

### AFTER YOU MERGE

1. Cut a release tag using the GitHub workflow. Wait for it to complete and the notification to be posted in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P


[CHI-2790]: https://tech-matters.atlassian.net/browse/CHI-2790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
